docs/resources/mlflow_experiments.md

```diff
@@ -28,6 +28,7 @@ resource "databricks_mlflow_experiment" "test" {
 
 The following arguments are supported:
 
-* `name` - (Required) Name of MLflow experiment.
+* `name` - (Required) Name of MLflow experiment. It must be an absolute path within the Databricks workspace, e.g. '/Users/\<some-username\>/my-experiment'. For more information about changes to experiment naming conventions, see 'https://docs.databricks.com/applications/mlflow/experiments.html#experiment-migration'.
```

Comment:
Would it be worth adding the language that "If artifact location is specified then it has to be an absolute path?"

Reply:
I just used the same standard error message received while creating the resource.